route4me-csharp-sdk/Route4MeSDKLibrary/Consts.cs

```diff
@@ -244,6 +244,10 @@ public static class R4MEInfrastructureSettingsV5
         public const string ContactsGetAsyncJobResult = ContactHost + "/job-tracker/result";
         public const string AddressBookDepots = ContactHost + "/addresses/depots";
 
+        public const string ContactExport = ContactHost + "/addresses/export";
+        public const string ContactExportByAreas = ContactHost + "/addresses/export-by-areas";
+        public const string ContactExportByAreaIds = ContactHost + "/addresses/export-by-area-ids";
```

Comment:
value already defined in ContactsExportByAreaIds field

Reply:
fixed, uploaded